_posts/2017-04-03-about-the-rust-community-team.md

```diff
@@ -0,0 +1,71 @@
+---
+
+The Rust Community Team is close to its second year of founding. At this point, I consider the dust has settled enough.
+
+In this post, I'll explain how we are set up and how you can follow our work.
+
+## A very short history
+
+The Rust Community IRC channel was created by me after feeling the need to get in touch with other Meetup organisers. I ran the [Berlin Rust Hack and Learn](...) and slowly, there were other events popping up.
+
+Shortly after launch of Rust 1.0, the Rust Community team was founded and [announced](https://internals.rust-lang.org/t/announcing-the-community-subteam/2248) by [Steve Klabnik]() as a way to bring representation of community workers into the Rust project. See then announcement for the initial goals. Later, [Erick Tryzelaar]() took over the lead. After 2 years of active work, we have at some points diverged from this and extended it, but many still hold true.
+
+The [roster](https://www.rust-lang.org/en-US/team.html#Community-team) has changed expanded since, especially with organisers of meetups and conferences joining.
+
+Most members of the team did community work in other communities, which certainly gave us a headstart in some aspects. Still, the only requirement to participate is interest in our work and participation.
+
+At the same time, the still-young Rust project was in a sense a blank slate, making it very easy to evolve the structure of community work: it was only just forming. For me, personally, that was a very refreshing experience.
+
+The community team is not the [❤️moderation team❤️](), although there is some personal overlap: helping out with discussions is not our 
+
+## Setup: We do FOSS community
+
+Community work for free and open source software often means a non-commercial environment. This comes with its own set of challenges and freedoms.
+
+The Rust Community team is a volunteer effort. Even the Mozilla employees on the team are volunteering next to their job. Obviously, there are sometimes connections to their day jobs.
+
+This, most importantly, means that the most constrained resource of the team is time. We have find relatively lightweight and easy to pick up processes. Still, documentation and coordination is important, which introduces work. We believe though, that - in total - we reduce work for individuals. It is the express goal of the team to be supportive to projects, not introduce more effort to them.
+
+Still, FOSS work is subject to the tides of life much more then a proper job. We all have our lives with ups and downs. While we try our best to be reliable, always keep this in mind should an interaction with us does not end up perfect or delayed. But still, we want to be measured by your expectations and not disappoint you.
+
+If you are a member of another FOSS community, treat our work with the same attitude you have to other projects code: with respect, but with the willingness to take, be inspired, ask for advice and adopt!
+
```

Comment:
There's another blog post of all tiny things coming up.( basically a port of my lobste.rs commend)